ansible/roles/dashmate/tasks/main.yml

```diff
@@ -231,52 +247,75 @@
   args:
     chdir: '{{ dashmate_source_dir }}'
   changed_when: build_dashmate_helper.rc == 0
+  when: dashmate_branch is defined
+
+- name: Get dashmate release assets
+  ansible.builtin.uri:
+    url: "https://api.github.com/repos/dashpay/platform/releases/tags/v{{ dashmate_version }}"
+    method: GET
+    status_code: 200
+    return_content: true
+  register: release_info
+  when: dashmate_version is defined
+
+- name: Get package architecture
+  ansible.builtin.command: dpkg --print-architecture
+  register: deb_architecture
+  when: dashmate_version is defined
+  changed_when: false
+
+- name: Install dashmate package
+  ansible.builtin.apt:
+    deb: "{{ release_info.json.assets | selectattr('name', 'search', '_' + deb_architecture.stdout + '.deb') | map(attribute='browser_download_url') | first }}"
```

Comment:
This fails on arm64 with:
```
TASK [dashmate : Install dashmate package] *********************************************************************************************************************
fatal: [hp-masternode-3]: FAILED! => 
  msg: |-
    The task includes an option with an undefined variable. The error was: No first item, sequence was empty.. No first item, sequence was empty.
  
    The error appears to be in '/home/strophy/dash-network-deploy/ansible/roles/dashmate/tasks/main.yml': line 267, column 3, but may
    be elsewhere in the file depending on the exact syntax problem.
  
    The offending line appears to be:
  
  
    - name: Install dashmate package
      ^ here
```

Reply:
That's wired. You probably use an old version that doesn't have arm64 package yet. https://github.com/dashpay/platform/releases/tag/v0.24.0-dev.23